doc/Programming-guide/Key-platform-features/Endpoint-ownership/index.md

```diff
@@ -10,53 +10,61 @@ sort_idx: 40
 * TOC
 {:toc}
 
-To exchange events between several endpoints, it is required that those endpoints were attached to the same owner.
-Depending on the application, owners may be person, groups of people, or organizations. 
-As an example we can take your mobile phone or smart watch. 
-These devices are your possessions (attached to you) and you are owner of these devices. 
-Another example is smart TV located in your house and all your family members are users of this device. 
-In this case your family is group of people (owner) that own this smart TV and smart TV is endpoint attached to this group.
-Kaa provides necessary APIs to attach/detach endpoints to/from owners through one of the following two flows:
+To enable exchanging events between your [endpoints]({{root_url}}Glossary/#endpoint-ep), you need to have those endpoints attached to the same owner.
```

Comment:
I don't like this as the rationale. Rather, in the events documentation it is worth mentioning that endpoints must be attached to the same user.
However, here you should start from first principles, explaining that endpoint ownership allows associating endpoints to owners (persons, households, etc.).